### DISCUSSION TITLE
Le Nombre de Membre dans une guilde

### DISCUSSION THREAD

**Discussion by Kennen69** [Crownicles/Crownicles#1965] [Nouvelles idées]
Au niveau des guildes, pourquoi ne pas faire évoluer le nombre de membres par rapports au niveau ?
exemple pour une guilde : 6 membres a la création et puis au niveau 30 de la guilde passer le nombre de membres à 10 ou 15 ?

**Comment by Nargacaura** [comment]
Me semble qu'il y a déjà la #598 qui traite du sujet de l'augmentation de la capacité d'une guilde, même si c'est d'une autre manière.
